driver-core/src/main/java/com/datastax/driver/core/ProtocolVersion.java

```diff
@@ -26,13 +26,14 @@
   V2("2.0.0", 2, V1),
   V3("2.1.0", 3, V2),
   V4("2.2.0", 4, V3),
-  V5("3.10.0", 5, V4);
+  V5("3.10.0", 5, V4),
```

Comment:
Are we comfortable with V5 reporting 3.10 as its min version? 

E.g. when refreshing the schema, the control connection will assume 3.10-like schema tables if the control host talks V5 but doesn't report any version. This doesn't look accurate to me, but maybe that's not really important after all.

Reply:
Sorry, this appears to be a leftover from when I was thinking we could bring v5 out of beta for both the 3.11 and 4.0 lines. I'll make 4.0.0 the min version for V5.